### ISSUE TITLE
Don't Decay the Learning Rate, Increase the Batch Size 

### ISSUE THREAD

**Issue by lockder** [OpenNMT/OpenNMT-tf#254]
Could be posible to implement this in some way?
https://openreview.net/forum?id=B1Yy1BxCZ

**Comment by guillaumekln** [comment]
In general, that's not possible to achieve because the batch size is already tuned so that the GPU memory is fully used, leaving no room to increase it. I don't see clear benefits in implementing that.

**Comment by lockder** [comment]
in my case I have plenty of memory, but if i set up a very big batch size, then it takes too long to train and the performance on test is not that good.

**Comment by lockder** [comment]
the other way i saw is in this link checking the part of multiple  gpu
https://aws.amazon.com/es/blogs/machine-learning/the-importance-of-hyperparameter-tuning-for-scaling-deep-learning-training-to-multiple-gpus/

but I'm not sure if this is possible either. having an small lr then increase it and after that lower it

**Comment by guillaumekln** [comment]
Increasing then lowering the learning rate is possible, and it is actually what is done with the `noam_decay` schedule.

In any case, this results apply to standard image models, not sure that it works similarly for sequence-to-sequence models.

**Comment by lockder** [comment]
okey, thank you!
